--- /var/tmp/portage/net-dns/opendnssec-2.1.1/work/opendnssec-2.1.1/Makefile.am	2017-05-01 16:03:24.243999791 +0000
+++ /tmp/Makefile.am	2017-05-01 16:04:43.727999784 +0000
@@ -36,8 +36,6 @@
 	$(INSTALL) @INSTALLATIONUSERARG@ @INSTALLATIONGROUPARG@ -d $(DESTDIR)$(localstatedir)/lib/opendnssec/signconf
 	$(INSTALL) @INSTALLATIONUSERARG@ @INSTALLATIONGROUPARG@ -d $(DESTDIR)$(localstatedir)/lib/opendnssec/unsigned
 	$(INSTALL) @INSTALLATIONUSERARG@ @INSTALLATIONGROUPARG@ -d $(DESTDIR)$(localstatedir)/lib/opendnssec/signed
-	$(INSTALL) -d $(DESTDIR)$(localstatedir)/run
-	$(INSTALL) @INSTALLATIONUSERARG@ @INSTALLATIONGROUPARG@ -d $(DESTDIR)$(localstatedir)/run/opendnssec
 
 docs:
 	(cd libhsm; $(MAKE) doxygen)
